db/linker.xml

```diff
@@ -1199,6 +1199,7 @@
        <target name="^.*dragonfly.*$" />
        <target name="^.*-fastos.*$" />
        <target name="^.*qnx[0-9]*$" />
+       <target name="^(i.86|x86_64)-gnu$" /><!-- Hurd -->
```

Comment:
Let's add the "Hurd" comment to line 1191, WDYT?

Reply:
Hello.
I see no benefit in duplicating the list, and two reasons not to do it.
When a regex is added, the comment must be kept in sync. This causes unnecessary merge conflicts (see above).
Most of the comment is unneeded duplication of information, the rest is unclear. For example, I guess `fastos` matches BSD systems, but this is only a guess after thinking about the paragraph as a whole.
I want future reader to immediately know for sure that I intend this apparently unrelated regex to match Hurd.

Reply:
OK, fair enough.